src/main/resources/db.scripts.mvp/R__Session_Template_Data.sql

```diff
@@ -5,241 +5,288 @@
+        ('BLI_G1','BLI','Visits Main Hall','SOCIAL',20,1,false,'14:00','15:00','2023-01-01','2023-01-01','SUNDAY',false),
+        ('BLI_G1','BLI','Visits Main Hall','SOCIAL',20,1,false,'15:30','16:30','2023-01-01','2023-01-01','SUNDAY',false),
+        ('BLI_G1','BLI','Visits Main Hall','SOCIAL',20,1,false,'14:00','15:00','2023-01-03','2023-01-03','TUESDAY',false),
+        ('BLI_G1','BLI','Visits Main Hall','SOCIAL',20,1,false,'15:30','16:30','2023-01-03','2023-01-03','TUESDAY',false),
+        ('BLI_G1','BLI','Visits Main Hall','SOCIAL',20,1,false,'14:00','15:00','2023-01-04',NULL,'TUESDAY',true),
+        ('BLI_G1','BLI','Visits Main Hall','SOCIAL',20,1,false,'15:30','16:30','2023-01-04',NULL,'TUESDAY',true),
+        ('BLI_G1','BLI','Visits Main Hall','SOCIAL',20,1,false,'14:00','15:00','2023-01-04',NULL,'WEDNESDAY',true),
+        ('BLI_G2','BLI','Visits Main Hall','SOCIAL',20,1,false,'15:30','16:30','2023-01-04',NULL,'WEDNESDAY',true),
+        ('BLI_G1','BLI','Visits Main Hall','SOCIAL',20,1,false,'14:00','16:00','2023-01-04',NULL,'FRIDAY',true),
+        ('BLI_G1','BLI','Visits Main Hall','SOCIAL',20,1,false,'14:00','15:00','2023-01-04',NULL,'SATURDAY',true),
+        ('BLI_G1','BLI','Visits Main Hall','SOCIAL',20,1,false,'15:30','16:30','2023-01-04',NULL,'SATURDAY',true),
+        ('BLI_G1','BLI','Visits Main Hall','SOCIAL',20,1,false,'14:00','15:00','2023-01-04',NULL,'SUNDAY',true),
+        ('BLI_G2','BLI','Visits Main Hall','SOCIAL',20,1,false,'15:30','16:30','2023-01-04',NULL,'SUNDAY',true),
+        ('BLI_G1','BLI','Visits Main Hall','SOCIAL',20,1,false,'14:00','15:00','2023-01-09',NULL,'TUESDAY',true),
+        ('BLI_G2','BLI','Visits Main Hall','SOCIAL',20,1,false,'15:30','16:30','2023-01-09',NULL,'TUESDAY',true),
+        ('BLI_G1','BLI','Visits Main Hall','SOCIAL',20,1,false,'14:00','16:00','2023-01-09',NULL,'WEDNESDAY',true),
+        ('BLI_G2','BLI','Visits Main Hall','SOCIAL',20,1,false,'14:00','16:00','2023-01-09',NULL,'FRIDAY',true),
+        ('BLI_G1','BLI','Visits Main Hall','SOCIAL',20,1,false,'14:00','15:00','2023-01-09',NULL,'SATURDAY',true),
+        ('BLI_G1','BLI','Visits Main Hall','SOCIAL',20,1,false,'15:30','16:30','2023-01-09',NULL,'SATURDAY',true),
+        ('BLI_G1','BLI','Visits Main Hall','SOCIAL',20,1,false,'14:00','15:00','2023-01-09',NULL,'SUNDAY',true),
+        ('BLI_G1','BLI','Visits Main Hall','SOCIAL',20,1,false,'15:30','16:30','2023-01-09',NULL,'SUNDAY',true),
+        (NULL,'CFI','Visits Main Hall','SOCIAL',35,0,false,'13:45','15:45','2023-01-23',NULL,'MONDAY',false),
+        (NULL,'CFI','Visits Main Hall','SOCIAL',35,0,false,'13:45','15:45','2023-01-23',NULL,'TUESDAY',false),
+        (NULL,'CFI','Visits Main Hall','SOCIAL',0,2,false,'13:45','14:45','2023-01-23',NULL,'TUESDAY',false),
+        (NULL,'CFI','Visits Main Hall','SOCIAL',15,0,true,'18:00','18:45','2023-01-23',NULL,'TUESDAY',false),
```

Comment:
The capacity of 15 (for enhanced) rather than 35 like the other open visit slots is going to break our workaround on the front-end of just assuming all slots at the same establishment have the same capacity 😬 - will raise on Jira!